oscar_odin/mappings/context.py

```diff
@@ -278,21 +278,23 @@ def bulk_update_or_create_many_to_many(self):
         m2m_to_create, m2m_to_update, _ = self.get_all_m2m_relations
 
         # Create many to many's
-        for relation, instances in m2m_to_create.items():
+        for relation, instances_to_create in m2m_to_create.items():
             fields = self.get_fields_to_update(relation.related_model)
             if fields is not None:
-                validated_m2m_instances = self.validate_instances(instances)
-                relation.related_model.objects.bulk_create(validated_m2m_instances)
+                if relation.related_model != self.Model:
```

Comment:
Can we assume that validate_resources is sufficient for product fields as this skips some custom clean methods of Product model?

Reply:
yes indeed I think so. For parents this was not an issue because of the structure (it tests different things) but for standalones they check if the title and slug are not none which we don't care about at this point.

Maybe we should only do this in the productmodelcontext and not in the normal modelcontext
like make it a settings on the context? What do you think

Reply:
yes, thats better